test/runner/allure_test.js

```diff
@@ -39,4 +43,34 @@ describe('CodeceptJS Allure Plugin', () => {
       done();
     });
   });
+
+  it('before suite errors are reported when running', (done) => {
+    exec(codecept_run_config('before_suite_test_failed.conf.js'), (err, stdout) => {
+      stdout.should.not.include('OK  | 0 passed');
+      stdout.should.include('FAIL  | 0 passed, 1 failed');
+
+      const files = fs.readdirSync(path.join(codecept_dir, 'output/failed'));
+      const testResultPath = files[0];
+      assert(testResultPath.match(/\.xml$/), 'not a xml file');
+      const file = fs.readFileSync(path.join(codecept_dir, 'output/failed', testResultPath), 'utf8');
+      file.should.include('failing setup test suite: BeforeSuite failed');
+      file.should.include('the before suite setup failed');
+      done();
+    });
+  });
+
+  it('before suite errors are reported when running in workers', (done) => {
+    exec(codecept_workers_config('before_suite_test_failed.conf.js'), (err, stdout) => {
+      stdout.should.not.include('OK  | 0 passed');
```

Comment:
I would say this assert is not needed :)

Reply:
good point, will change in next commit

Reply:
https://github.com/Codeception/CodeceptJS/pull/2248/commits/69c6c960420ec3b32894e1c4bb6afe4cd7b4e23d